### PR TITLE
Update and cleanup executors docstrings

### PR DESCRIPTION
**Pull request by RafyAmgadBenjamin**
### Description

Add a docstring to ```RemoteExecutor``` class 

### Issue
https://github.com/threefoldtech/js-ng/issues/380

### CODE REVIEW

File: jumpscale/core/executors/remote.py [hunk 1]
```diff
@@ -35,6 +35,17 @@ def execute(cmd, command_ctx, connection_ctx):
 
 
 class RemoteExecutor:
+    """Executor remote allows executing commands within specific env on the local machine. using the executor framework you can retrieve the stdout, stderr, and the return code as well.
```
Comment (abom) [inline jumpscale/core/executors/remote.py:38]: Maybe using "remote machine" here is better?

```suggestion
    """Remote executor allows executing commands within specific env on the remtote machine. using the executor framework you can retrieve the stdout, stderr, and the return code as well.
```

File: jumpscale/core/executors/remote.py [hunk 2]
```diff
@@ -35,6 +35,17 @@ def execute(cmd, command_ctx, connection_ctx):
...
+    e.g
+        with j.core.executors.RemoteExecutor(host="localhost", connect_kwargs={"key_filename":"/home/xmonader/.ssh/id_rsa",}) as c:c.run("hostname")
+
+    Args:
+        connection_ctsx(dict): context passed
```
Comment (abom) [inline jumpscale/core/executors/remote.py:43]: I think what's required in #380 is more explicit info about possible options that can be passed, maybe we can list the most important ones at least?

Comment (RafyAmgadBenjamin) [reply jumpscale/core/executors/remote.py]: done

File: jumpscale/core/executors/remote.py (hunk 1 above)
Comment (RafyAmgadBenjamin) [reply jumpscale/core/executors/remote.py]: done

Comment (RafyAmgadBenjamin) [reply jumpscale/core/executors/remote.py]: done

**Review by ranatrk — APPROVED** [review]

File: jumpscale/core/executors/remote.py [hunk 3]
```diff
@@ -35,6 +35,17 @@ def execute(cmd, command_ctx, connection_ctx):
 
 
 class RemoteExecutor:
+    """Remote remote allows executing commands within specific env on the remote machine. using the executor framework you can retrieve the stdout, stderr, and the return code as well.
+    e.g
+        with j.core.executors.RemoteExecutor(host="localhost", connect_kwargs={"key_filename":"/home/xmonader/.ssh/id_rsa",}) as c:c.run("hostname")
+
+    Args:
+        connection_ctx(dict): context passed to fabric e.g. fabric.Connection(host, user=None, port=None, config=None, gateway=None, forward_agent=None, connect_timeout=None, connect_kwargs=None, inline_ssh_env=None)
```
Comment (xmonader) [inline jumpscale/core/executors/remote.py:43]: can you add what keys expected in connect_kwargs too?

Comment (RafyAmgadBenjamin) [reply jumpscale/core/executors/remote.py:43]: Done

**Review by waleedhammam — APPROVED** [review]

**Pull request merged**